Added Nagvis and missing graph widget

albundy83 opened the pull request.

Commit:
Added Centreon Nagvis module installation

Commit:
Install MK Livestatus + Nagvis

Commit:
Added Widget Graph URL and installation

Commit:
Added missing WIDGET_GRAPH_VER var

Commit:
Added Nagvis module version

Commit:
Added to fix a bug in Nagvis authentication save

Here the reference link :
http://sugarbug.web4me.fr/atelier/techniques/ihmweb/cartographie_supervision/centreon-nagvis/

Commit:
Update www-data user to centreon-engine group

 To allow access to /var/lib/centreon-engine/rw/live for Nagvis

zeysh commented:
Hi albundy83,

Thanks for your contribution.
I need some delay for testing your PR.

Bye

Commit:
Upgraded versions to latest available

Upgraded versions and removed the php5.3 installation as it's not needed anymore :
https://documentation.centreon.com/docs/centreon/en/latest/release_notes/centreon-2.6.0.html

But a manual modification is needed due to a bug in formGenerateTraps.php :

/usr/local/centreon/www/include/configuration/configGenerateTraps/formGenerateTraps.php

line : 149                         if (isset($ret["generate"]["generate"]) && $ret["generate"]["generate"]) {
and 
line : 167                          if (isset($ret["apply"]["apply"]) && $ret["apply"]["apply"] &&

must be updated with :

line : 149                         if (isset($ret['generate']) && $ret['generate']) {
and 
line : 167                         if (isset($ret['apply']) && $ret['apply'] &&

Thanks to : 
http://www.sugarbug.web4me.fr/atelier/installations/debian/debian_centreon_26/

Commit:
Updated versions

Upgraded version to reflect the one in sript.
Removed not need / at the end of some paths.

Commit:
Update README.asciidoc

Commit:
Upgraded versions

Commit:
Updated versions

Commit:
Added support for Debian Jessie, apache 2.4 and MariaDB 10.1

Commit:
Updated MariaDB version

Commit:
Bumped versions to latest release

Commit:
Bumped versions to latest release

Commit:
Updated nagios plugins

https://nagios-plugins.org/nagios-plugins-2-1-1-released/

Commit:
Bumped Nagios plugins version

Commit:
Fixed issue when extracting widgets

Tar file tree has changed

Commit:
FIxed missing library for centreontrapd

Thanks @george-vieira

albundy83 commented:
Latest release and updates.

Commit:
Update centreon.sh

Commit:
Reverted an easy password for people using find and replace

Commit:
Fixed error with Graph Monitoring widget

Here the error message :
PHP Fatal error: Call to undefined method DB_Error::fetchRow() in /usr/local/centreon/www/class/centreonGMT.class.php on line 298

Commit:
Bumped versions to latest release

Commit:
Merge pull request #1 from GreenCom-Networks/centreon-web-2.7

Initial test for centreon-web-2.7

Commit:
Added comment about new centreon-web interface

Commit:
Fixed missing space

Commit:
Update to latest widgets

Latest widgets fix bug introduce with latest Centreon web 2.7 release

Commit:
Bumped versions to align with script

Commit:
Updated to add latest updates

- bumped to Centreon Web 2.7.1
- added new Centreon widgets (Widgets Live Top 10 CPU, Widgets Live Top 10 Memory, Widgets Engine Status, Widgets Grid Map, Widgets HTTP-Loader, Widgets tactical overview)
- removed old code which were not used anymore (php 5.3 installation)
- commented the clapi installation as it's not used any more since Centreon Web 2.7.0 (it's not embedded in Centreon Web, defaut path is /usr/local/centreon/bin/centreon)
- minor fixes

Commit:
Bumped versions to latest release to follow source

Commit:
Fixed typo

Commit:
Fixed php pear install

- Some pear modules are not installed correctly (it will also prepare for possible php 7.0 support),
- Added apache centreon load configuration for debian jessie,
- Added modification for date.timezone in /etc/php5/apache2/php.ini

Commit:
Changed a2ensite to a2enconf

Commit:
Added apt upgrade after mariadb installation

To upgrade mysql-common

Commit:
Added -y option for upgrade

Commit:
Added chmod on generateSqlLite file

Commit:
Added Centreon and Broker templates

From a very good idea of @george-vieira
https://github.com/george-vieira/centreon-install/commit/b58dc4ef5a7fe4fa8ee50ac33b4800733b54a949

Commit:
Fixed templates generation and copy

Commit:
Fixed CENTREON_ENGINE_CONNECTORS in WEB_TMPL

Commit:
Updated to add latest features

Commit:
Added details about usage

Commit:
Fixed centreon web in comment

Commit:
Merge pull request #2 from GreenCom-Networks/new-templates

Added Centreon and Broker templates

Commit:
Update README.asciidoc

Commit:
some sort of merge

Commit:
added open_files_limit higher limits for mysql service

Commit:
Added: Allow MYSQL_PASSWORD env variable
Added: Error message when MYSQL_PASSWORD not set
Added: Role based installation with usage page
Updated: Version update ENGINE_VER='1.7.0', BROKER_VER='3.0.3', CENTREON_VER='2.8.4'

Commit:
Adjusted README for new changes

Commit:
Update README.asciidoc

Commit:
Added missing centreon-broker var path

Commit:
Centreon dockerfile

Commit:
Create Dockerfile

albundy83 committed:
Merge pull request #9 from george-vieira/master

feature updates
